### ISSUE TITLE
Define a brand image and logo

### ISSUE THREAD

**Issue by sbrl** [tldr-pages/tldr#2986]
GitHub have recently added a new feature to the repository settings known as _social preview_:

![image](https://user-images.githubusercontent.com/9929737/57339427-7dd1a580-7129-11e9-8978-86f5ef0c8a94.png)

[Here's the associated blog post](https://github.blog/2019-04-17-custom-open-graph-images-for-repositories/).

Should tldr-pages have a social preview image? If so, what image or screenshot should we use?

**Comment by sbrl** [comment]
Yeah, my design skills are somewhat limited too. I guess we could use the [image on the homepage](https://raw.githubusercontent.com/tldr-pages/tldr/master/screenshot.png) as an interim solution?

**Comment by agnivade** [comment]
Interesting. We have never really thought of any brand image or logo for tldr. I have zero design skills, but if somebody does, I am game for providing input.

**Comment by waldyrious** [comment]
There's a nice graphic in [this blog post](https://laravel-news.com/tldr-pages) about tldr pages. I'm not sure if it's more useful than the screenshot in our homepage and README, but it is certainly less busy and easier to read. Maybe the post's author, @paulredmond, can offer it for us to use?

**Comment by paulredmond** [comment]
> There's a nice graphic in this blog post about tldr pages. I'm not sure if it's more useful than the screenshot in our homepage and README, but it is certainly less busy and easier to read. Maybe the post's author, @paulredmond, can offer it for us to use?

We use https://thenounproject.com/ for icons and Eric Barnes owns the copyright for content on Laravel-news.com. My advice when considering using icons for branding/marks be careful to read the TOS in how you can use them.

**Comment by Aracki** [comment]
I think the graphic @waldyrious mentioned is more appropriate than the picture with a lot of commands. No one will actually try to read something from social preview image?

I can help with a design/redesign.

**Comment by waldyrious** [comment]
Thanks for the clarification and advice @paulredmond. I couldn't find that specific icon in the Noun Project, but in any case I agree that if we're to use this for our own branding, we might want to design the icon ourselves. @Aracki if you could make a proposal, that'd be great :) something simple, similar to the graphic linked above would work well IMO.

**Comment by Aracki** [comment]
![image](https://user-images.githubusercontent.com/6129986/61576955-29f81b00-aae1-11e9-8fca-ae9cd8bcf02f.png)

Proposals here ^^

First one is for the social preview, but I suggest we open an extra Issue for changing the beginning of the main README file, which will include a small version of the logo.

**Comment by agnivade** [comment]
Great stuff ! I like the gray and teal scheme better. I noticed that there are some transparency issues around the edges in the small logos. Is that intended ?

Also, what is that purple icon on top of the >_ icon for in the main preview ?

**Comment by Aracki** [comment]
1. I am not sure about which transparency issue you are referring to.
2. A purple icon will not be visible though. (I took the screenshot)

I think you all can view the previews on [this link](https://www.figma.com/file/YRKudGV9FcGF3p0XFG17C8/Untitled?node-id=0%3A1).

All suggestions and critics are welcome. (@waldyrious , @sbrl , @pxgamer, @mebeim , @schneiderl and others as well) 🙂

**Comment by mebeim** [comment]
Nice work @Aracki. I think the transparency issue @agnivade is talking about is this: 

![image](https://user-images.githubusercontent.com/14198070/61578173-253c6280-aaf3-11e9-900f-d358208d2321.png)

but it only looks like an editing artifact that would be gone in the final image, am I right?

By the way, I definitely like the blue background version better: 

![image](https://user-images.githubusercontent.com/14198070/61578199-5fa5ff80-aaf3-11e9-93fb-3bfa6861b3a5.png)

I think this would be cool to have as a social card!

As per the square icons, I'm not entirely sure about them, the logo seems too small compared to the rest. I still like the blue version more though.

**Comment by andrik** [comment]
Great work @Aracki 

As @mebeim I prefer the blue version

**Comment by sbrl** [comment]
Hey, awesome work @Aracki!

I'm fine with either version - they both look ok to me! Though I'm always an advocate for more colour, so I'd probably say I prefer blue (though the grey looks a bit like a terminal, so it's ok too).

We could potentially update the favicon on tldr.sh too if we all agree on the new logo.

**Comment by mebeim** [comment]
Wait a second though, now that I think about it: the current TLDR logo is composed of these two characters: `$;`

Could we get a preview of that, with the same format shown above? Basically just `$;` instead of `>_`, I would like to see how that looks (sorry but I'm not that practical with the website you linked).

**Comment by Aracki** [comment]
@mebeim The logo itself was drawn by my brother. I am just putting it together with different backgrounds. 😕

* About the favicon, maybe it would be better to have it without text. My favorite is the **third version**.

* About the social preview, I have added two more examples. **third blue version** I preffer over the **first version**, as it is little bit colder and goes better with the logo. 

[The same link with the new drawings](https://www.figma.com/file/YRKudGV9FcGF3p0XFG17C8/Untitled?node-id=0%3A1).

**Comment by agnivade** [comment]
Favicon looks great.

For the previews, I like 1st and 3rd both. The 1st one stands out a bit more due to more contrast. But 3rd is fine too.

**Comment by waldyrious** [comment]
I like the designs too, and would be fine with either `$;` or `>_` (the former evokes a shell CLI a but more to my eye, though), and with either color scheme.

One thing I'd point out is that we should make sure we use a freely-licensed font, and possible something more legible than the highly square one (especially for the tagline). One of the [monospaced fonts in Google Fonts](https://fonts.google.com/?category=Monospace), for instance, would be a good choice. Perhaps [Share Tech Mono](https://fonts.google.com/specimen/Share+Tech+Mono), or [Major Mono Display](https://fonts.google.com/specimen/Major+Mono+Display).

I'd also not use fully justified alignment in the tagline, which makes the word spacing irregular across the two lines. Left-aligned would be fine IMO.

**Comment by schneiderl** [comment]
Wow nice work @Aracki 
The blue background one looks awesome.

**Comment by sbrl** [comment]
Perhaps even _Source Code Pro_ would be an option? Share tech looks ok, but I'm not too sure about major mono.

**Comment by Aracki** [comment]
But AFAIK font converted to vector is not a text or font anymore. It becomes a vector drawing which you can safely use in your logo. Correct me if I'm wrong. I am saying this because I really prefer the old logo versions. PS. The font name from which the vector is made is [Bank Gothic](https://en.wikipedia.org/wiki/Bank_Gothic).

I would definitely choose _Source Code Pro_ if we are going with Google Fonts. It's in the first four examples. 

[The same link with new examples](https://www.figma.com/file/YRKudGV9FcGF3p0XFG17C8/TLDR?node-id=0%3A1).

**Comment by mebeim** [comment]
I had a go at this myself. In the end I think I definitely prefer the `>_` version, but I'm unsure about the current colors. I kind of like it more dark to be honest :thinking: 

 `$;` | `>_`
-|-
![tldr-dollar256](https://user-images.githubusercontent.com/14198070/62411186-d660f380-b5ef-11e9-8c52-b2400cc5d85d.png) | ![tldr-dash-256](https://user-images.githubusercontent.com/14198070/62411047-87ff2500-b5ee-11e9-96cf-d0db662ab125.png)
![tldr-dollar-cream256](https://user-images.githubusercontent.com/14198070/62411256-9a7a5e00-b5f0-11e9-963a-3062df3c831c.png) | ![tldr-dash-cream256](https://user-images.githubusercontent.com/14198070/62411260-9f3f1200-b5f0-11e9-988b-8977291ca389.png)
![tldr-dollar-cyan256](https://user-images.githubusercontent.com/14198070/62411285-e6c59e00-b5f0-11e9-99d3-8b0153439209.png) | ![tldr-dash-cyan256](https://user-images.githubusercontent.com/14198070/62411261-a5cd8980-b5f0-11e9-98a5-dbe716378931.png)

**Comment by agnivade** [comment]
Personally, I think the dark backgrounds by @Aracki were fine. And my vote is with >_ too.

**Comment by sbrl** [comment]
Nice, @mebeim! Yeah, a darker background would work better I think.

`>_` is my preferred option, too

**Comment by waldyrious** [comment]
Just out of curiosity, for those who prefer `>_`, what's your motivation for that choice?

I personally think `$_` could work well, since AFAIK `$` is the most common shell prompt, and `_` seems appropriate to represent a blinking cursor that's expecting user input; both together would then convey the notion of a terminal prompt where the user is expected to input command lines.

But this is just a digression -- I don't mean to derail the discussion, just to understand your preferences :)

**Comment by agnivade** [comment]
The options were $; and >_. Among the two, >_ seemed better. But I am find with $_ too.

**Comment by andrik** [comment]
I agree with @waldyrious 

I am not a big fan of `; ` because reminds me programming languages

**Comment by Aracki** [comment]
Should we make a poll for choosing the final option?

And what others think of font usage [discussion](https://github.com/tldr-pages/tldr/issues/2986#issuecomment-516082795)?

**Comment by agnivade** [comment]
Could you show us a sample of how `$_` looks ? I think that would help us come to a decision.

Regarding fonts, I like Ubuntu Mono too. Though I will leave it up to you for a final decision.

**Comment by sbrl** [comment]
Yeah, I think `$_` would be fine too

**Comment by Aracki** [comment]
I don't have "designer powers" to do that on my own.

About which we agreed in the end? Can we used `Bank Gothic` font or we need to go with `Source Code Pro`? What colors the majority prefer?

**Comment by mebeim** [comment]
So here's a little table of the alternatives from which to choose and the current status (from what I could see from above comments):

| Button text | Button color | Button text color | BG color | Font |
|-|-|-|-|-|
`>_`, `$;`, `$_` | Cyan, Dark blue, [other?] | Dark blue, White, [other?] | Dark blue, Grey, [other?] | Bank Gothic, S.C. Pro, [other?] |
| TBD: `>_` or `$_` | TBD | TBD | Dark blue | TBD |

Where "TBD" = To Be Decided. 

For the font I would say S.C. Pro but really any decent looking and free to use monospace font does it for me. Another important thing to decide is: **do we also want the name inside the logo or not?** I personally am not a big fan of having the name repeated in the logo.

**Comment by agnivade** [comment]
Thanks for reviving this thread. The options on this [comment](https://github.com/tldr-pages/tldr/issues/2986#issuecomment-516082795) are great.

If it is hard to see how `$_` looks, I would suggest to go ahead with one of the options on that comment and come back later to refine it. It is certainly better to have some preview rather than having no preview.

I like the name-less icon in the blue and teal scheme. I think the majority was in favor of the blue and teal scheme. A logo should stand on it's own without needing the name inside it.

**Comment by agnivade** [comment]
@andrik @schneiderl @pxgamer - what do you think about moving ahead with what we have currently rather than waiting to see other options ?

**Comment by pxgamer** [comment]
@agnivade, I think that sounds good 👍

**Comment by andrik** [comment]
> @agnivade, I think that sounds good

Me too

**Comment by schneiderl** [comment]
@agnivade I say we go ahead with it. As you said, it is better to have some preview rather than having no preview. And I actually think the options presented in here are great :)

**Comment by waldyrious** [comment]
I just re-read the entire conversation and tried to gauge the opinions expressed in it. Just to make sure we're on the same page:

If we used this image for the icon/logo/favicon:

<img src="https://user-images.githubusercontent.com/478237/67817735-7ff83980-faa6-11e9-8352-d7035518045b.png" width="200"/>

...and this one for social preview:

![](https://user-images.githubusercontent.com/14198070/61578199-5fa5ff80-aaf3-11e9-93fb-3bfa6861b3a5.png)

...with the following changes (everything else being equal, colors etc.):
- Font changed to [Source Code Pro](https://fonts.google.com/specimen/Source+Code+Pro)
- Icon text changed from `>_` to `$_`
- Slogan changed from justified to left-aligned

...would that be acceptable to everyone? Or 

If so, I'd be happy to recreate this in SVG so that we can wrap up the discussion :)

**Comment by mebeim** [comment]
Sounds fine to me @waldyrious

**Comment by agnivade** [comment]
@waldyrious - Do you have the color codes ?

**Comment by waldyrious** [comment]
No, I was planning on reconstructing those with Inkscape. I think that work is worth it because it's important that we retain the vector source of the image (so that we can make variations at different sizes, colors, etc. as needed).

**Comment by mebeim** [comment]
@waldyrious just to save you some time, [here are my SVGs](https://www.dropbox.com/s/xztyz1mvq606oyc/tldr-logos.tar.gz?dl=0) if they can be of any help.

**Comment by sbrl** [comment]
Yep, sounds great to me @waldyrious :smiley_cat:

**Comment by waldyrious** [comment]
@mebeim the SVG was superb, thanks! So here's my proposal based on the described above:

Logo:
<img src="https://user-images.githubusercontent.com/478237/68078761-d03c0800-fdd4-11e9-8b23-d65195a8d7cc.png" width="200"/>

Social preview:
![tldr-logotype svg](https://user-images.githubusercontent.com/478237/68078830-602e8180-fdd6-11e9-97f5-4ffa356ab873.png)


Note: I used Liberation Mono and Ubuntu Mono for the text, because Source Code Pro's dollar sign deviates from the traditional shape by not having the vertical bar cross the entire "S".

I can make further adjustments if you guys want. Once we settle on a definitive design, I'll submit the SVG and PNG files to the repo.

**Comment by waldyrious** [comment]
By the way, I was thinking we could take the opportunity to replace the logo for tldr-bot, to follow the same general design. Something like this, perhaps?

![rect4175](https://user-images.githubusercontent.com/478237/68078954-92d97980-fdd8-11e9-824f-50bdd8e7aff5.png)

**Comment by mebeim** [comment]
Hmmm... I definitely like the `>_` version more. That `$` sign does not even seem like a fixed-width font :thinking:. The images from [your previous comment](https://github.com/tldr-pages/tldr/issues/2986#issuecomment-547679559) are way better IMHO.

**Comment by agnivade** [comment]
Thanks for the work @waldyrious ! All looks good to me. I am fine with both `$_` and `>_`.

**Comment by Aracki** [comment]
For me `>_` is a way better looking. If we are going with `Source Code Pro` the second one from the [link](https://www.figma.com/file/YRKudGV9FcGF3p0XFG17C8/TLDR?node-id=0%3A1) is the best option in my opinion.

PS. @waldyrious I really like the `tldr-bot` logo.

**Comment by waldyrious** [comment]
I agree that the "$" could look more code-y. Could you guys try out the following text:

```
TL;DR

$_
```

in https://app.programmingfonts.org/, using the largest available font size (32), and let me know which one(s) you prefer? Here are some examples:

| <img width="133" alt="tldr Agave" src="https://user-images.githubusercontent.com/478237/68084109-36eb1100-fe29-11e9-95d1-7fd698a31840.png"> | <img width="125" alt="tldr APL2741" src="https://user-images.githubusercontent.com/478237/68084110-36eb1100-fe29-11e9-868b-025079dd9ef0.png"> | <img width="120" alt="tldr BorgSansMono" src="https://user-images.githubusercontent.com/478237/68084111-3783a780-fe29-11e9-9176-37b349cd1ac9.png"> | <img width="121" alt="tldr Inconsolata" src="https://user-images.githubusercontent.com/478237/68084112-3783a780-fe29-11e9-80a7-665472a62564.png"> |
|-|-|-|-|
| Agave | APL2741 | Borg Sans Mono | Inconsolata |
| <img width="132" alt="tldr Profont" src="https://user-images.githubusercontent.com/478237/68084113-3783a780-fe29-11e9-8077-3881a4051266.png"> | <img width="118" alt="tldr SometypeMono" src="https://user-images.githubusercontent.com/478237/68084114-381c3e00-fe29-11e9-80f9-8cb0b33d88d6.png"> | <img width="110" alt="tldr UbuntuMono" src="https://user-images.githubusercontent.com/478237/68084219-add4d980-fe2a-11e9-8b42-24cab414667b.png"> | <img width="106" alt="tldr VictorMono" src="https://user-images.githubusercontent.com/478237/68084115-381c3e00-fe29-11e9-84ac-3afe4c779958.png"> |
| Profont | Sometype Mono | Ubuntu Mono | Victor Mono |

**Comment by mebeim** [comment]
Profont, Inconsolata and Sometype look better than the others to me.

**Comment by andrik** [comment]
I really liked Sometype and APL2741 versions

BTW, awesome work @waldyrious

**Comment by Aracki** [comment]
I prefer a simpler one as **_Sometype_** and **_Inconsolata_**. Both really nice.

**Comment by agnivade** [comment]
Inconsolata/VictorMono/SometypeMono for me.

**Comment by sbrl** [comment]
Seems like the common thread here is everyone likes _Sometype Mono_, so perhaps we could go with that?

Awesome work @waldyrious :D

**Comment by ostera** [comment]
Loving this thread <3 will be cool to update tldr.sh (and the web client) to use the same theme once it's out :)

**Comment by waldyrious** [comment]
Alright guys! Thanks for the feedback and the patience. Here are the updated designs incorporating the suggestions you made. All pieces of text are now using [Sometype Mono](https://github.com/dharmatype/Sometype-Mono) as the font. Let me know what you think!

Logo/icon:
<img alt="tldr-logo svg" src="https://user-images.githubusercontent.com/478237/68551101-4e199800-0401-11ea-82e9-4828e69c5370.png" width="200"/>

Social preview:
![tldr-logotype svg](https://user-images.githubusercontent.com/478237/68551100-4e199800-0401-11ea-9a62-8c10d952a6af.png)

**Comment by waldyrious** [comment]
I just realized that we can actually add a period to the end of the tagline, which would make it perfectly justified without resorting to any weird spacing :D

WDYT?

![tldr-logotype svg](https://user-images.githubusercontent.com/478237/68551182-03e4e680-0402-11ea-92f3-5f4b1ea20eea.png)

**Comment by mebeim** [comment]
The button bevel looks too narrow now that I notice it :thinking:. Other than that looks fine.

**Comment by agnivade** [comment]
Looks fine to me :+1:

**Comment by schneiderl** [comment]
That looks great. 
Thank you guys for this 😄

**Comment by waldyrious** [comment]
I don't think the bevel looks too narrow. Why do you say that?

I can change the text in the button to be bold, though, if there are no objections.

**Comment by pxgamer** [comment]
I think it looks great as is!

**Comment by Aracki** [comment]
Maybe I would thicken up `TL;DR` just a little bit. But it looks very nice like this. Great idea to add a dot @waldyrious. 👍

**Comment by sbrl** [comment]
Yeah, the bevel looks fine, and nice one with the justification! 

Let's use this!

Shall I apply the logo and the 2nd social preview?

**Comment by agnivade** [comment]
@mebeim - can you respond to the bevel concerns ? :)

**Comment by mebeim** [comment]
> I don't think the bevel looks too narrow. Why do you say that?

No reason other than personal preference, really. Keycaps have a very specific shape, if you carefully observe what they look like from above. IMHO a wider bevel (and also decentered) better recalls the shape of a keycap. 

Examples:

![image](https://user-images.githubusercontent.com/14198070/68772093-7f74ac80-0629-11ea-961e-80ccbf46209d.png)
![image](https://user-images.githubusercontent.com/14198070/68772109-86032400-0629-11ea-9799-a2f07d4cb300.png)

**Comment by sbrl** [comment]
Is it supposed to look like a key on a keyboard? It's got multiple characters on it, so I think it looks fine as-is.

**Comment by agnivade** [comment]
I guess it's meant to look like a key. But IMO, it looks fine as is. If @waldyrious feels like spending more time on this, then I guess it's okay, otherwise I would suggest to go with what we have. And then come back to this when somebody has more free time.

@waldyrious - I also think we should upload the icon assets somewhere so that if folks want to experiment on the icon in future, they can ?

**Comment by sbrl** [comment]
Just let me know when to set the icon / social preview etc.

**Comment by waldyrious** [comment]
I get where you're coming from, @mebeim. I just think that with the variety of hardware we have today, especially with laptops that use very flat keycaps, it's not a big deal that we don't depict the exact shape of old-school keyboards. The current design feels sufficiently generic to me to fit the notion of a keycap without trying to emulate a particular style. Besides, as @sbrl said, we do need the space for the actual label of the key, especially since the logo is meant to be usable in very small sizes.

> @waldyrious - I also think we should upload the icon assets somewhere so that if folks want to experiment on the icon in future, they can ?

Yes, that was my plan from the start! 😃 As I said [above](https://github.com/tldr-pages/tldr/issues/2986#issuecomment-547808019): "it's important that we retain the vector source of the image (so that we can make variations at different sizes, colors, etc. as needed)."

@sbrl if the images above are the right size / aspect ratio, as far as I'm concerned you can go ahead and set them up in the social preview settings (make sure you use the banner with the final period that justifies the text 😄). I will clean up the SVG and submit a PR shortly, which once merged will finally close this issue! 🎉

**Comment by agnivade** [comment]
Woot ! profile picture for our org is set ! Check out - https://github.com/tldr-pages. (Might need a ctrl+shift+r)

**Comment by waldyrious** [comment]
I've left some screenshots in #3606, please take a look!

**Comment by sbrl** [comment]
Replied in the PR @waldyrious

**Comment by waldyrious** [comment]
> Perhaps we can close this issue now?

I just opened #3619 which should close this once it's merged :)

By the way, I wonder who controls our Twitter account [@tldr_pages](https://twitter.com/tldr_pages). Was it @igorshubovych or am I misremembering? We should update the account's profile picture (and maybe think about reviving the account, e.g. by setting up some automated posting of random examples from the repo, or something like that :))

**Comment by sbrl** [comment]
Ah, that's a good idea @waldyrious. Maybe the tldr-bot could be updated to do that.

**Comment by waldyrious** [comment]
https://github.com/tldr-pages/tldr-bot/pull/8 :)
